modules/EmergencyMedicine/src/EmergencyMedicine/Controller/letterGeneratorController.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+namespace EmergencyMedicine\Controller;
+/**
+ *
+ * This is a template class for develop and testing
+ *
+ */
+class letterGeneratorController extends BaseController
```

Comment:
this file contains old functions you used for tests, let's remove it.
It contains functions from vaccines vertical, it is confusing...

Reply:
As I said I wanted to keep it so it can be used as a template for other letters